front-end/src/components/player-page/stats/leaderboard-ranking.tsx

```diff
@@ -0,0 +1,30 @@
+import {Tooltip, Typography} from "@mui/material";
+import React from "react";
+import {getNumberWithSuffix} from "../../../common/utils";
+
+interface LeaderboardRankingProps {
+    rank: number
+    leagueAvg: number
+    totalPLayersOnLeaderboard: number
+}
+
+const FIRST = "#FFA500";
```

Comment:
Refactor these out into a more global file. Tbh this should be part of the theme.